x-pack/test/lists_api_integration/common/config.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import path from 'path';
+import { CA_CERT_PATH } from '@kbn/dev-utils';
+import { FtrConfigProviderContext } from '@kbn/test/types/ftr';
+import { services } from './services';
+
+interface CreateTestConfigOptions {
+  license: string;
+  disabledPlugins?: string[];
+  ssl?: boolean;
+}
+
+export function createTestConfig(name: string, options: CreateTestConfigOptions) {
+  const { license = 'trial', disabledPlugins = [], ssl = false } = options;
```

Comment:
Since these are defaulted in `x-pack/test/lists_api_integration/security_and_spaces/config.ts` does it need to be defaulted here too?

Reply:
No, from a technical standpoint it does not, it just follows the other patterns where a default is given and then everything still overrides it. I will probably keep it for the sake of the other examples in the other plugins doing it the same way but fwiw.

Later if we introduce tests only for basic license rather than trial which gives us full platinum access then that config.ts would look like this:

```ts
export default createTestConfig('security_and_spaces', { license: 'basic' });
```